<launch>
    <!-- height min, height max, width min, width max -->
    <arg name="adj_cnst_list" default="[0.55, 0.9, 0.5, 1.0]"/>

    <node name="pub_seg_sample_2d" pkg="sitl_dvrk" type="pub_seg_sample_2d.py" output="screen">
        <param name="calib_dir"             type="str"     value="L2R"/>
        <param name="num_pts_cnt"           type="int"     value="100"/>
        <param name="num_pts_bnd"           type="int"     value="20"/>
        <param name="adj_dist_upp_bnd"      type="int"     value="100"/>
        <rosparam param="adj_cnst_list" subst_value="True"> $(arg adj_cnst_list) </rosparam>
    </node>

    <node name="pub_seg_sample_3d"  pkg="sitl_dvrk" type="pub_seg_sample_3d.py" output="screen">
        <param name="window_size"           type="int"     value="15"/>
        <param name="window_thr"            type="double"  value="3e-4"/>
    </node>

    <?ignore
    
    ?>
</launch>